monitor/grafana/zeebe.json

```diff
@@ -10979,7 +10979,7 @@
                 "uid": "${DS_PROMETHEUS}"
               },
               "editorMode": "code",
-              "expr": "sum(rate(atomix_append_entries_data_rate_total{cluster=~\"$cluster\", namespace=~\"$namespace\", partition=~\"$partition\"}[$__rate_interval])) by (partition, follower)",
+              "expr": "sum(rate(atomix_append_entries_data_rate_total{cluster=~\"$cluster\", namespace=~\"$namespace\", partition=~\"$partition\"}[$__rate_interval]) or rate(atomix_append_entries_data_rate_seconds_total{cluster=~\"$cluster\", namespace=~\"$namespace\", partition=~\"$partition\"}[$__rate_interval])) by (partition, follower)",
```

Comment:
:question: This is a counter, right? Why are we adding `_seconds` here? Is it because it's the base unit?

Reply:
Same for the one below.

Reply:
Seems like we don't specify units. So this seems wrong to me :x:

Reply:
Removed it, did one too many :-D